libs/eo/certificates/src/lib/eo-certificates.component.ts

```diff
@@ -27,20 +29,14 @@ import { EoCertificatesTableComponent } from './eo-certificates-table.component'
     WattCheckboxComponent,
     EoCertificatesTableComponent,
     WattButtonComponent,
+    WattCardComponent,
```

Comment:
Why is WattCardComponent added?

Reply:
Removed. check latest commit:

[6bc893d](https://github.com/Energinet-DataHub/greenforce-frontend/pull/1967/commits/6bc893d766cf2c001a8977e1f087b77e2e8e38a7)